tests/unittests/Foundation/ReferenceFoundation/TestNSArray.mm

```diff
@@ -150,13 +150,19 @@ static int objectIndexInArray(NSArray* array, int value, int startingFrom, int l
     //        NSBinarySearchingOptions* searchForBoth = NSBinarySearchingFirstEqual | NSBinarySearchingLastEqual
     //        auto _ = objectIndexInArray(array, 9, 0, 13, searchForBoth)
 
-    auto notFound = objectIndexInArray(array, 11, 0, 13);
-    ASSERT_EQ_MSG(notFound, NSNotFound, @"NSArray return NSNotFound if object is not found.");
```

Comment:
We'd like to avoid making edits to this file, in order to stay mapped exactly to the original test at [https://github.com/apple/swift-corelibs-foundation/blob/master/TestFoundation/TestNSArray.swift](https://github.com/apple/swift-corelibs-foundation/blob/master/TestFoundation/TestNSArray.swift). Could you test these cases in a separate test case in tests/unittests/Foundation/NSArrayTests.mm, and revert this file to its original state?